src/api/utils/template/templates.py

```diff
@@ -58,12 +62,13 @@ def validate_template(template):
     )[0]
 
     try:
-        rn = random.randint(0, 5000)
+        # Ignore this bandit error, not used for security/cryptography purposes.
```

Comment:
For improved context, please include the name/id of the bandit error in question here.

Reply:
I have provided a link in the comments for B311 - random. https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b311-random